src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -2,15 +2,21 @@
 
 import com.smartcar.sdk.data.*;
 
+import javax.json.Json;
+import javax.json.JsonObject;
+
 import okhttp3.FormBody;
 import okhttp3.HttpUrl;
 import okhttp3.Request;
 import okhttp3.RequestBody;
+import okhttp3.MediaType;
 
 /**
  * Smartcar Vehicle API Object
  */
 public class Vehicle extends ApiClient {
+  public static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
```

Comment:
Not sure if this should be in `Vehicle.java` or `ApiClient.java` or if there's a constants file that would be better for this.

Reply:
`ApiClient.java` would probably make the most sense.